sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/LongCounterSdk.java

```diff
@@ -35,15 +34,15 @@ public void add(long increment) {
   }
 
   @Override
-  BoundInstrument newBinding(InstrumentAccumulator instrumentAccumulator) {
-    return new BoundInstrument(instrumentAccumulator);
+  BoundInstrument newBinding(InstrumentProcessor instrumentProcessor) {
+    return new BoundInstrument(instrumentProcessor);
```

Comment:
this definitely doesn't feel right to me. The Accumulator should be handling the binding, so it can keep track of when to released the resources associated with the labelset. I don't think that's the job of the Processor, which needs to hold long-term state of all the labelsets against which recordings have been made, even if they're not currently being actively recorded.

Reply:
It is not, as explained in the other comment. I think you are keep reviewing this.